packages/app-security/src/components/SecureRoute.tsx

```diff
@@ -1,24 +1,21 @@
 import * as React from "react";
-import { hasScopes, hasRoles } from "@webiny/app-security";
+import { hasScopes } from "@webiny/app-security";
 import { getPlugin } from "@webiny/plugins";
 import { ResourcesType } from "../identity";
 import { SecureRouteErrorPlugin } from "@webiny/app-security/types";
 
 export default ({
     children,
-    scopes,
-    roles
+    scopes
 }: {
     children: any;
     scopes?: ResourcesType;
-    roles?: ResourcesType;
 }): React.ReactElement => {
     const checks = {
-        scopes: scopes ? hasScopes(scopes, { forceBoolean: true }) : true,
-        roles: roles ? hasRoles(roles, { forceBoolean: true }) : true
+        scopes: scopes ? hasScopes(scopes, { forceBoolean: true }) : true
```

Comment:
Clean this area please. We had the `checks` object here just because there were `scopes` and `roles` props available. Not we only have `scopes`.